docs/components/plugins/api/APISectionTypes.tsx

```diff
@@ -117,9 +119,7 @@ const renderType = ({ name, comment, type }: TypeGeneralData): JSX.Element | und
   } else if (type.types && (type.type === 'union' || 'intersection')) {
```

Comment:
Actual check evaluates either to `falsy` value (coming from `type.types`) or to `true || 'intersection'`. I believe you want something like this:
```suggestion
  } else if (type.types && ['union', 'intersection'].includes(type.type)) {
```

Reply:
Yup, I have missed brackets in there, but your suggestion is a cleaner way, thanks! 🚀